.travis.yml

```diff
@@ -17,7 +17,6 @@ matrix:
         - pip freeze
       script:
         - flake8 openslides tests
-        - isort --check-only --diff --recursive openslides tests
```

Comment:
It makes travis faster.

If isort sorts the imports the same of python 3.6 and 3.7, there is no need to check it twice. If it sorts them differently, then we have to choose one sorting (3.6 or 3.7) I would suggest, that we always use the sorting of the newest python version. Therefore only the newest python version has to check isort.

The same is true for the auto formatting of black.

Actually, we should also remove the flake8 check from python 3.6. mypy should stay on all supported python versions, because the return value of a function could change in a python version and we have to make sure, that we support both types.